Cargo.toml

```
@@ -18,15 +24,15 @@ include = [
     "/LICENSE-MIT",
     "/LICENSE-APACHE",
 ]
-edition = "2018"
+edition = "2021"
 
 [dependencies]
-embedded-hal = "0.2.5"
+embedded-hal = "1.0.0"
 smbus-pec = "1"
 
 [dev-dependencies]
-linux-embedded-hal = "0.3"
-embedded-hal-mock = "0.9"
+linux-embedded-hal = "0.4"
+embedded-hal-mock = { version = "0.10", features = ["eh1"] }
```

```suggestion
embedded-hal-mock = { version = "0.10", default-features=false, features = ["eh1"] }
```
To avoid e-h 0.2 dependency.